src/ContainerDetails.jsx

```diff
@@ -12,6 +12,14 @@ const render_container_state = (container) => {
     return cockpit.format(_("Exited"));
 };
 
+const render_container_published_ports = (container) => {
```

Comment:
Style hint: For presentational components you can write `({ prop1, prop2 }) =>` which makes the supported properties more explicit.